quiz.css

```diff
@@ -0,0 +1,80 @@
+/*Validated using css validator, no errors found*/
+
+/*overall styling*/
+* {
+	box-sizing: border-box;
+}
+
+.main-content {
+	max-width: 1020px;  
+	margin: 0 auto;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+/*card styling*/
+
+article.card {
+	float: left; /*v2: change this area using media queries for mobile-responsive layout. currently this layout only works on screens about 720px and wider.*/
+	width: 30%;
+	border: 1px solid lightblue;
+	margin: 5px 1.666666667%;
+}
+
+article > header { 
+	border: 1px dotted lightgrey;
+	text-align: center;
+	font-size: .8em;
+}
+
+/*SECTION 1 (information) STYLING*/
+.product-image {
+	display: block;
+	margin: 15px auto;
+	width: 150px;
+	height: 150px; /*updated to have image sizes in CSS not HTML per comment*/
+}
+
+
+div.product-description {
+	height: 190px;
+}
+
+div.product-description p {
+  font-size: 0.85em;
+  padding: 10px;
+  text-align: justify;
+}
+
+.product-availability {
+	background-color: darkgrey;
+	color: yellow;
+	width: 100%;
+	margin: 8px 0;
+	font-size: .8em;
+	line-height: 1.5em;
+}
+
+/*sections 2 and 3 styling. V2 refactor to be one section? or at least have the same class? */
```

Comment:
As to your comment, definitely.  Since you've got three different rulesets all applied to the same two classes, your best bet is to just apply them to a single class and put that class on  both types of section.